src/fidesops/ops/models/privacy_request.py

```diff
@@ -71,35 +70,27 @@
 
 
 class ManualAction(BaseSchema):
-    """Surface how to manually retrieve or mask data in a database-agnostic way
+    """Surface how to retrieve or mask data in a database-agnostic way
 
     "locators" are similar to the SQL "WHERE" information.
     "get" contains a list of fields that should be retrieved from the source
-    "update" is a dictionary of fields and the value they should be replaced with.
+    "update" is a dictionary of fields and the replacement value/masking strategy
```

Comment:
Reusing a structure created for the "manual" connector to cache details regarding further actions needed by some other party to continue processing of this request.